data/json/items/comestibles/irradiated_fruit.json

```diff
@@ -4,7 +4,7 @@
     "id": "irradiated_rose_hips",
     "copy-from": "rose_hips",
     "name": { "str_sp": "irradiated rose hips" },
-    "description": "Irradiated rose hips.",
+    "description": "An irradiated rose hips.",
```

Comment:
"rose hips" is plural (you can see that the name is used for both possessives), so the old version is actually the correct one, I think. You could do something like `Some irradiated rose hips.` and have it be correct, though.

Reply:
Actually, I think half of the contents if this file is incorrect. Berries copy-from their corresponding berry entries, and those entries are for a standard 250 ml unit of berries, not a single berry, while the copy-from fruit do seem to be for a single fruit (with a volume based on the fruit). Thus, the file should be updated to match the underlying entries, resulting in one pattern for (individual) "fruit" (bananas are actually berries...) and one for standard units of "berries".

I wouldn't use "some" for entries that refer to a standard unit of whatever, as that's too vague. It indicates that you'd be able to stack them like e.g. some sugar or some detergent (i.e. adding two "somes" would result in a "some" of a larger volume), rather than being one standard unit. Thus, for rose hips I'd prefer the original description.

Reply:
I'm open for suggestions! Indeed, these entries are best to be updated, especially after #56361. Feel free to commit or point any